### PR TITLE
Chapter 11: Account activation

### PR DESCRIPTION
**Pull request by datbk206**
image
Link kiến thức cần nắm chương 11:
https://docs.google.com/document/d/1TqCZ0KuFaEIyt21az404GWBtCdk5lQfgDkkgI0PaR6k/edit?usp=sharing

### CODE REVIEW

**Comment by datbk206** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -35,16 +35,32 @@ def remember
     update_attribute :remember_digest, User.digest(remember_token)
   end
 
-  def authenticated? remember_token
-    BCrypt::Password.new(remember_digest).is_password? remember_token
+  def authenticated? attribute, token
+    digest = send "#{attribute}_digest"
+    return false if digest.nil?
```
Comment (ghost) [inline app/models/user.rb]:
```suggestion
    return false unless digest
```

**Review by ghost — COMMENTED** [review]
cmt

**Comment by datbk206** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**